### PR TITLE
Chore: Add Shuttle.toml description

### PR DESCRIPTION
**Pull request by sentinel1909**
Good morning,

I added a new section to the docs called "Configuration" and within that a new article called "Shuttle Service Name".  This article covers the purpose of Shuttle.toml.  Please let me know what you think.  Will make any edits/changes if needed.

:)

Thanks,
~Jeff Mitchell

### CODE REVIEW

File: configuration/shuttle-service-name.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Shuttle Service Name"
+---
+
+The name of a Shuttle service (your project), can be set up front as part of the ```cargo shuttle init``` command. In the series of setup questions, the name is the first to be chosen.  Later, to change the name, a configuration file called ```Shuttle.toml``` can be created in the root of the project folder, at the same level as ```Cargo.toml```.  In this file, simply put: ```name = <your-project-name>```. As soon as the ```Shuttle.toml``` file exits, it will control the name of the Shuttle service and override other choices.
```
Comment (oddgrd) [inline configuration/shuttle-service-name.mdx:5]: For in-line code blocks like this you can use just a single backtick, three backticks are only needed for multi-line code blocks.

**Review by oddgrd — COMMENTED** [review]
Hey Jeff, thanks again, this looks great! :pray: I would also add a bit more information on how the name can be set. So it goes like this: If the `--name <name>` arg is passed to a shuttle command, that will be used over Shuttle.toml or Cargo.toml package name. If a Shuttle.toml exists, it will be used over Cargo.toml. If none of the above, the Cargo.toml package name will be used.

File: configuration/shuttle-service-name.mdx (hunk 1 above)
Comment (sentinel1909) [reply configuration/shuttle-service-name.mdx:5]: Sounds good.  Will make the adjustments!

**Comment by sentinel1909** [comment]
Ok, have updated as suggested above.  I also took the liberty of fixing a typo I saw in the FAQ entry regarding Shuttle.toml. I also made a minor edit to point to the revised article in the new Configuration section.  Looks like I might have created merge conflict by doing that though...oops. :(

**Comment by oddgrd** [comment]
Thanks, the changes look good! A merge conflict can be daunting, but it's usually not as scary as it looks. It just means that someone else made changes to main after you created your branch, and their changes conflict with yours. So to merge your branch into main, you need to decide whether to keep your or their change (or a combination). Here is an article on git merge conflicts: https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts

For this particular conflict, what you would need to do is merge main into your branch, then choose which change to keep for that conflict, which in this case would be your change. Then push the changes to your branch like normal. It takes a while to get used to git, but take your time and you'll get there. And if you need help with this, feel free to reach out to me on discord!

**Comment by sentinel1909** [comment]
Alright!  I think I did it.  Seems like it's all clean now.  I abandoned my proposed changes to the FAQ, can redo it later if needed.

File: support/faq.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ This section is aimed at collecting common questions users have to provide docum
     Yes, you can create a SeaORM connection from the sqlx pool Shuttle passes to you when you provision a SQL database. You can take a look at the example [here](https://github.com/shuttle-hq/shuttle/issues/179#issuecomment-1203536025).
   </Accordion>
   <Accordion title="Can I rename my shuttle project?">
-    Certainly. Just create a `Shuttle.toml` file in the folder where your `Cargo.toml` is and add `name = "your-project-name`.
+    Certainly. Just create a `Shuttle.toml` file in the some folder where your `Cargo.toml` is and add `name = "your-project-name`.
```
Comment (oddgrd) [inline support/faq.mdx:33]:
```suggestion
    Certainly. Just create a `Shuttle.toml` file in the same directory where your `Cargo.toml` is and add `name = "your-project-name"`.
```

**Pull request merged**